docs/dictionary/property/executionContexts.lcdoc

```diff
@@ -25,7 +25,7 @@ Value:
 The <executionContexts> is similar to a call stack, it consists of a
 list of contexts, one per line, with the most recent context at the end.
 Each context is a string of the form:
-&lt;object-long-id&gt;,&lt;handler-name&gt;,&lt;line-number&gt; 
+&lt;object-long-id&gt;,&lt;handler-name&gt;,&lt;line-number&gt;,&lt;behavior-object-long-id&gt;
```

Comment:
@mwieder should we wrap the behavior in `[]` to clarify it might not be there? Perhaps this would be better as a list:
```
Each context is a string of the following comma-delimited items:

1. object long id
2. handler name
3. line number
4. behavior object long id if the context is from a behavior script
```